Framework/API/src/ExperimentInfo.cpp

```diff
@@ -187,23 +187,27 @@ void checkDetectorInfoSize(const Instrument &instr,
                              "instrument");
 }
 
-boost::shared_ptr<Beamline::ComponentInfo>
-makeComponentInfo(const Instrument &oldInstr,
-                  const API::DetectorInfo &detectorInfo,
-                  std::vector<Geometry::ComponentID> &componentIds) {
+boost::shared_ptr<Beamline::ComponentInfo> makeComponentInfo(
```

Comment:
I did this to keep it consistent to the storage policy for `Beamline::DetectorInfo`. presumably this was driven by the need to facilitate copies.